zerver/webhooks/taiga/view.py

```diff
@@ -4,10 +4,10 @@
 
 *Emojis*: most of the events have specific emojis e.g.
 - :notebook: - change of subject/name/description
-- :chart_with_upwards_trend: - change of status
+- :chart: - change of status
 etc. If no there's no meaningful emoji for certain event, the defaults are used:
-- :thought_balloon: - event connected to commenting
```

Comment:
I think I agree with Tim, let's remove all the emojis. None of our other integrations use emojis in their notifications, so this seems highly non-standard. :)

Reply:
Do you want me to completely remove the emojis from the integration? I mean, the emojis should not be there, when testing manually using the payload?